platforms/minigame/platforms/taobao-mini-game/wrapper/builtin/window.js

```diff
@@ -13,23 +13,10 @@ export cancelAnimationFrame from './cancelAnimationFrame'
 export * from './WindowProperties'
 
 
-const compareVersions = (curVersion, supportedVersion) => {
-    const curVersionNum = curVersion.split('.').map(Number);
-    const supportedVersionNum = supportedVersion.split('.').map(Number);
-
-    for (let i = 0; i < Math.max(curVersionNum.length, supportedVersionNum.length); i++) {
-        const curVersionPart = curVersionNum[i] || 0;
-        const supportedVersionPart = supportedVersionNum[i] || 0;
-        if (curVersionPart > supportedVersionPart) return 1;
-        if (curVersionPart < supportedVersionPart) return -1;
-    }
-    return 0;
-}
-
 import MultiWebSocket from './MultiWebSocket'
 import WebSocket from './WebSocket'
 const systemInfo = my.getSystemInfoSync();
-if (compareVersions(systemInfo.version, '10.35.30') >= 0) {
+if (typeof my !== 'undefined' && my && my.canIUse && my.canIUse('connectSocket.object.multiple')) {
```

Comment:
Is typeof my !== 'undefined' needed?

Reply:
![image](https://github.com/cocos/cocos-engine/assets/3078423/fb8b3644-c6fe-4129-b94b-9cd7b9ce02d4)

Reply:
Can just use like
```
if (my?.canIUse?.('connectSocket.object.multiple')) {
} else {
}
```

Reply:
report error on Chrome
![image](https://github.com/cocos/cocos-engine/assets/3078423/f314b1c8-0b64-4b23-8e9a-4540bb041494)